src/main/java/com/patchworkmc/manifest/converter/GloomDefenitionParser.java

```diff
@@ -0,0 +1,50 @@
+package com.patchworkmc.manifest.converter;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+
+import io.github.fukkitmc.gloom.definitions.ClassDefinition;
+import io.github.fukkitmc.gloom.definitions.GloomDefinitions;
+import io.github.fukkitmc.gloom.definitions.SelfMember;
+
+import com.patchworkmc.manifest.accesstransformer.AccessTransformerEntry;
+import com.patchworkmc.manifest.accesstransformer.AccessTransformerList;
+
+public class GloomDefenitionParser {
+	private GloomDefenitionParser() {
+		// NO-OP
+	}
+
+	public static GloomDefinitions parse(AccessTransformerList list) {
+		HashMap<String, Set<SelfMember>> classFields = new HashMap<>();
```

Comment:
I think we should store this data in a class and use a single HashMap here. Perhaps call it `DefinitionBuilder` or something descriptive.